net/usbwlan.c

```diff
@@ -0,0 +1,521 @@
+/*
+ * Phoenix-RTOS
+ *
+ * USB WLAN driver
+ *
+ * Copyright 2025 Phoenix Systems
+ * Author: Julian Uziembło
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/minmax.h>
+#include <sys/threads.h>
+#include <unistd.h>
+
+#include <posix/utils.h>
+
+#include <usbdriver.h>
+#include <usbhost.h>
+
+#include "sys/msg.h"
+#include "usb.h"
```

Comment:
```suggestion
#include <sys/msg.h>
#include <usb.h>
```

nit but maybe move inclusion of `sys/msg.h` to the highest include block